spec.html

```diff
@@ -40393,6 +40496,90 @@ <h1>`Promise.allSettled` Reject Element Functions</h1>
+        <p>The `any` function returns a promise that is fulfilled by the first given promise to be fulfilled, or rejected with an `AggregateError` holding the rejection reasons if all of the given promises are rejected. It resolves all elements of the passed iterable to promises as it runs this algorithm.</p>
+        <emu-alg>
+          1. Let _C_ be the *this* value.
+          1. Let _promiseCapability_ be ? NewPromiseCapability(_C_).
+          1. Let _promiseResolve_ be GetPromiseResolve(_C_).
+          1. IfAbruptRejectPromise(_promiseResolve_, _promiseCapability_).
+          1. Let _iteratorRecord_ be GetIterator(_iterable_).
+          1. IfAbruptRejectPromise(_iteratorRecord_, _promiseCapability_).
+          1. Let _result_ be PerformPromiseAny(_iteratorRecord_, _C_, _promiseCapability_, _promiseResolve_).
+          1. If _result_ is an abrupt completion, then
+            1. If _iteratorRecord_.[[Done]] is *false*, set _result_ to IteratorClose(_iteratorRecord_, _result_).
+            1. IfAbruptRejectPromise(_result_, _promiseCapability_).
+          1. Return Completion(_result_).
+        </emu-alg>
+        <emu-note>
+          <p>The `any` function requires its *this* value to be a constructor function that supports the parameter conventions of the `Promise` constructor.</p>
+        </emu-note>
+
+        <emu-clause id="sec-performpromiseany" aoid="PerformPromiseAny">
+          <h1>Runtime Semantics: PerformPromiseAny ( _iteratorRecord_, _constructor_, _resultCapability_, _promiseResolve_ )</h1>
+          <p>The abstract operation PerformPromiseAny takes arguments _iteratorRecord_, _constructor_, _resultCapability_ (a PromiseCapability Record), and _promiseResolve_. It performs the following steps when called:</p>
+          <emu-alg>
+            1. Assert: ! IsConstructor(_constructor_) is *true*.
+            1. Assert: ! IsCallable(_promiseResolve_) is *true*.
+            1. Let _errors_ be a new empty List.
+            1. Let _remainingElementsCount_ be a new Record { [[Value]]: 1 }.
+            1. Let _index_ be 0.
+            1. Repeat,
+              1. Let _next_ be IteratorStep(_iteratorRecord_).
+              1. If _next_ is an abrupt completion, set _iteratorRecord_.[[Done]] to *true*.
+              1. ReturnIfAbrupt(_next_).
+              1. If _next_ is *false*, then
+                1. Set _iteratorRecord_.[[Done]] to *true*.
+                1. Set _remainingElementsCount_.[[Value]] to _remainingElementsCount_.[[Value]] - 1.
+                1. If _remainingElementsCount_.[[Value]] is 0, then
+                  1. Let _error_ be a newly created `AggregateError` object.
+                  1. Perform ! DefinePropertyOrThrow(_error_, `"errors"`, Property Descriptor { [[Configurable]]: *true*, [[Enumerable]]: *false*, [[Writable]]: *true*, [[Value]]: ! CreateArrayFromList(_errors_) }).
+                  1. Return ThrowCompletion(_error_).
+              1. Return _resultCapability_.[[Promise]].
```

Comment:
I think you dropped a level of indentation here and on subsequent lines: this should be a substep of the `If _next_ is *false*, then` bit, and the `1. Let _nextValue_ be IteratorValue(_next_)` bit should be within the `Repeat`, not following it.